### PR TITLE
realsense camera description

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

**Comment by ipa-nhg** [comment]
@ipa-fmw should I also open a pull request with these changes to ipa320 ?? what do you think?

File: cob_description/urdf/sensors/realsense.gazebo.xacro [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  <xacro:macro name="realsense_gazebo" params="name ros_topic">
+    <gazebo reference="${name}_link">
+      <sensor type="depth" name="${name}_depth_frame">
+        <always_on>true</always_on>
+        <update_rate>25.0</update_rate>
+        <camera>
+          <horizontal_fov>${74.0*M_PI/180.0}</horizontal_fov>
+          <image>
+            <format>R8G8B8</format>
+            <width>320</width>
```
Comment (ipa-fmw) [inline cob_description/urdf/sensors/realsense.gazebo.xacro]: is this resolution correct? in the datasheet: https://software.intel.com/en-us/realsense/devkit it's listed as 640x480

File: cob_description/urdf/sensors/realsense.gazebo.xacro [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="realsense_gazebo" params="name ros_topic">
+    <gazebo reference="${name}_link">
+      <sensor type="depth" name="${name}_depth_frame">
+        <always_on>true</always_on>
+        <update_rate>25.0</update_rate>
+        <camera>
+          <horizontal_fov>${74.0*M_PI/180.0}</horizontal_fov>
```
Comment (ipa-fmw) [inline cob_description/urdf/sensors/realsense.gazebo.xacro]: is 74deg horizontal opening angle correct?

**Comment by ipa-fmw** [comment]
and yes, please open a PR to ipa320 too

File: cob_description/urdf/sensors/realsense.gazebo.xacro (hunk 1 above)
Comment (ipa-nhg) [inline cob_description/urdf/sensors/realsense.gazebo.xacro]: I have to adapt the simulation, I am waiting until the camera is mounted on the robot and I can test it, to adapt the simulation to the real behavior. All the  simulation parameters have to be reviewed

**Comment by ipa-nhg** [comment]
ok, I will do as soon as it is tested on the robot! :)

**Pull request merged**